app/views/nominees/ChooseNomineeView.scala.html

```diff
@@ -0,0 +1,61 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.nominees.routes._
+@import models.Mode
+@import views.ViewUtils._
+@import config.FrontendAppConfig
+
+@this(
+    govukLayout: templates.GovukLayoutWrapper,
+    formHelper: FormWithCSRF,
+    errorSummary: components.errorSummary,
+    inputRadio: components.inputRadio,
+    button: components.button,
+)
+
+@(form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
+
+@govukLayout(pageTitle = Some(title(form, "chooseNominee.title", Some("officialsAndNominees.section")))) {
+
+    @formHelper(action = ChooseNomineeController.onSubmit(mode)) {
+
+    @errorSummary(form.errors)
+
+    @inputRadio(
+        form = form,
+        legend = messages("chooseNominee.heading"),
+        inline = true,
+        items = Seq(
+            RadioItem(id = Some("value"),
```

Comment:
Could this be updated to "value-yes" if we are defining the ids this way please? (Will require a change in acceptance tests)